lib/server.go

```diff
@@ -924,12 +924,6 @@ func (srv *Server) _handleHeaderBundle(pp *Peer, msg *MsgDeSoHeaderBundle) {
 		len(msg.Headers), srv.blockchain.chainState(), pp,
 		srv.blockchain.headerTip().Header.Height, printHeight)))
 
-	// If the node is running a Fast-HotStuff validator and the consensus is running,
```

Comment:
The steady state FastHotStuffConsensus shouldn't preclude a validatorfrom accepting a header bundle from a peer. If the validator has fallen behind and it receives a header bundle from its sync peer, it should be able to process that header bundle and try to catch up as best it can